src/StackShared/FrameStackNavigator.cs

```diff
@@ -189,8 +201,12 @@ async Task<Page> UINavigate()
 						var viewTcs = new TaskCompletionSource<Page>();
 						page.Loaded += OnPageLoaded;
 
+#if WINUI
```

Comment:
I don't think you need this `#if` if you remove don't fully qualify the parameter name.